scripts/data-retention-remove-supplier-declarations.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+Our data retention policy is that users have accounts deactivated and personal data stripped after 3 years without
+a login.
+
+This script is very simple and has not been upgraded to accept any arguments to prevent the possibility of accidental
+deletion. If you are in doubt use the dry run option.
+
+Usage: data-retention-remove-user-data.py <stage> [--dry-run] [--verbose]
+
+Options:
+    --stage=<stage>                                       Stage to target
+
+    --dry-run                                             List account that would have data stripped
+    --verbose
+    -h, --help                                            Show this screen
+
+Examples:
+    ./scripts/data-retention-remove-user-data.py preview
+    ./scripts/data-retention-remove-user-data.py preview --dry-run --verbose
```

Comment:
The examples and the usage will need updating with the script name, otherwise docopt will complain 😸